### PR TITLE
Installation doc redraft

### LINKED ISSUE #1731: Add trouble shooting entry for "Library not loaded" error on OSX
OSX app seems to not open with a "Library not loaded: @rpath/Electron Framework.framework/Electron Framework" error when the downloaded zip is extracted with certain third-party apps like 'The Unarchiver' or 'iZip'. 

This has a known workaround where the app would work as expected if extracted via the built-in 'Archive Utility'.

### LINKED ISSUE #1773: Suggested Addition to Linux Installation Docs
> There is a bunch of config for Linux.  Currently, only have 1 posted community example, but can add more as we come across them.

There are some examples added in issues, should add to docs

### LINKED ISSUE #1960: Provide installation instructions after user downloads the app
Reported originally here: https://github.com/postmanlabs/postman-app-support/issues/7581

After downloading Postman for Linux, we don't communicate well to the users about what should be done next.

Slack shows platform-specific instructions post download.
### Ubuntu
image
### Fedora
image

### PR DESCRIPTION
**Pull request by SueSmith**
Generic redraft of installation guide, content hasn't really changed, just restructured etc - I did remove some of the troubleshooting section on specific problems, thinking if it's a temporary issue it probably belongs in the forum rather than docs..

I _think_/_hope_ this fixes #1960, fixes #1773, and fixes #1731 - also referencing postmanlabs/postman-app-support#7581

### CODE REVIEW

**Comment by SueSmith** [comment]
fyi just noticed #1960 so will flesh out linux instructions

File: src/pages/docs/postman/launching-postman/installation-and-updates.md [hunk 1]
```diff
@@ -24,221 +24,129 @@ contextual_links:
...
+>
+> Make sure you have read/write permission for the `~/.config` folder where Postman stores information.
+>
+> If you are an Ubuntu 18 user, you will also need to install the `libgconf-2-4` package to ensure a smooth Postman run: `apt-get install libgconf-2-4`
 
-   1. Make sure you have read/write permission for ```~/.config``` folder where Postman stores the information
+## Postman Chrome app (deprecated)
 
-   1. If you are an Ubuntu 18 user, you must additionally install libgconf-2-4 package to ensure a smooth Postman run. This package, shipped by default until Ubuntu version 18, has been dropped, and is mandatory for Postman to run. Use the following command to install ```libgconf-2-4```:
+The Postman Chrome app is deprecated—if you're using the Chrome app, you can retain your data when you [switch to the native app](https://blog.getpostman.com/2017/03/14/going-native/). The native app is built on [Electron](https://electronjs.org/), and overcomes a number of restrictions of the Chrome platform.
```
Comment (claireriley) [inline src/pages/docs/postman/launching-postman/installation-and-updates.md]: We may want to emphasize you'll only be able to retain your data **if you are logged in**. Similar to last line of the blog post. I'm sure we still get support tickets about it 😕 

> For people who are not signed in to their Postman account or who do not have sync enabled, you can export your data and import it into the new native app.

Comment (SueSmith) [reply src/pages/docs/postman/launching-postman/installation-and-updates.md]: Oh great thanks will add that 👍

Comment (SueSmith) [reply src/pages/docs/postman/launching-postman/installation-and-updates.md]: ok shifted this around a bit to really emphasise how to retain your data..

**Comment by SueSmith** [comment]
Added more detail on linux and osx unzip error.

**Comment by SueSmith** [comment]
@claireriley I've made those changes you requested, if you can review I think we can get this one merged.. 🤞

**Comment by SueSmith** [comment]
Oh good thanks for checking @kaustavdm

**Pull request merged**